### DISCUSSION TITLE
Default atom value = null then unable to set to string? Type Error

### DISCUSSION THREAD

**Discussion by Eduard-Hasa** [pmndrs/jotai#1636] [Q&A] [answered]
I am trying to create a atom that stores the string id of a selected item but by default want it to be null. Is this not possible? Im getting the following error:

Argument of type 'string' is not assignable to parameter of type 'SetStateAction<null>'.ts(2345)

**Comment by dai-shi** [comment, marked as answer]
You need to annotate or assert types.
```ts
const atom1 = atom<string | null>(null)
const atom2 = atom(null as string | null)
```

**Reply by Eduard-Hasa** [reply]
Thanks 😄
